tests/test_pythia8_fill.py

```diff
@@ -0,0 +1,100 @@
+import chromo
+from chromo.common import EventData
+from .util import run_in_separate_process
+import pytest
+import sys
+
+pytestmark = pytest.mark.skipif(
+    sys.platform == "win32", reason="Pythia8 does not run on windows"
+)
+
+
+def init_pythia8():
+    config = ["ProcessLevel:all = off" "ParticleDecays:tau0Max = 1e100"]
+    pythia8 = chromo.models.Pythia8(seed=1, config=config, banner=False)
+    return pythia8._pythia
```

Comment:
Why is this a function? It looks like you use this code exactly once. So I would inline it for clarity.

Reply:
Removed